src/Control/Director.php

```diff
@@ -2,6 +2,7 @@
 
 namespace SilverStripe\Control;
 
+use SilverStripe\Assets\Filesystem;
```

Comment:
Minor concern: increased coupling to `silverstripe/assets`. Looking at `SilverStripe\Assets\Filesystem`, a lot of the contents of that class aren’t really specific to our asset model, so it might be more appropriate to have that “helper” class in framework given how regularly we are/will be using it.

Can be separate to this PR and target 5.x of course :)

Reply:
I could move it to a `SilverStripe\Core` class maybe.